### PR TITLE
refactor: css color variable

### PR DESCRIPTION
**Pull request by kush-x7**
## Made CSS variables of colors in file variables.css and used those variables in all CSS files.
Resolved issue number https://github.com/Real-Dev-Squad/website-my/issues/308

### CODE REVIEW

File: app/styles/variables.css [hunk 1]
```diff
@@ -1,10 +1,163 @@
...
+
+  --button-get-started--bg:#041484;
+  --button-get-started--text: white;
+
+  --button-copy--bg: wheat;
+  --button-copy--text: #e30062;
+  --button-copy--border: #e30062;
+
+  --button-verify--text: white;
+  --button-identity--text: white;
```
Comment (vinayak-trivedi) [inline app/styles/variables.css:22]: I can see that a lot of variables are repeated, you can make it a bit generic so that you don't have to do that

Comment (kush-x7) [reply app/styles/variables.css:22]: Check this RFC https://github.com/Real-Dev-Squad/website-my/issues/311 Second approach.
I made changes according to it. Rohan approved the second approach.

Comment (vinayak-trivedi) [reply app/styles/variables.css:22]: okay, makes sense, thanks

**Review by vinayak-trivedi — APPROVED** [review]
LGTM

File: app/styles/image-upload.css [hunk 2]
```diff
@@ -59,7 +59,7 @@
 .image-preview {
   max-height: 500px;
   border-radius: 10px;
-  border: solid black;
+  border: solid var(--image-preview--border);
```
Comment (rohan09-raj) [inline app/styles/image-upload.css:62]: On line 105, we are using the same color as a border, why not reuse it?
It's used for border, right?, we don't have to care for which component it is used

File: app/styles/image-upload.css [hunk 3]
```diff
@@ -5,7 +5,7 @@
 }
 
 .drop-area {
-  border: 2px dashed #ccc;
+  border: 2px dashed var(--drop-area-border);
```
Comment (rohan09-raj) [inline app/styles/image-upload.css:8]: Refer to this comment
https://github.com/Real-Dev-Squad/website-my/pull/321/files#r1030068003

File: app/styles/navbar.css [hunk 4]
```diff
@@ -23,15 +23,15 @@ nav li {
...
+  color: var(--nav--link--hover);
 }
 
 .nav-menu {
   list-style-type: none;
   margin: 0;
   padding: 0;
   overflow: hidden;
-  background-color: #1d1283;
+  background-color: var(--nav-menu--bg);
```
Comment (rohan09-raj) [inline app/styles/navbar.css:34]: On line 9, this color is already used as navbar background, can we please reuse it ?

**Review by rohan09-raj — APPROVED** [review]

**Pull request merged**